app/controllers/stories_controller.rb

```diff
@@ -370,7 +370,11 @@ def set_article_json_ld
         "@id": URL.article(@article)
       },
       "url": URL.article(@article),
-      "image": ApplicationController.helpers.article_social_image_url(@article),
+      "image": [
+        ApplicationController.helpers.article_social_image_url(@article, width: 1080, height: 1080),
```

Comment:
@juliannatetreault  nice one, I've been learning more about SEO through your PR's :D 

On that note, some thoughts on this:
When using this helper would it not be useful to enforce the ratio through a maths calculation rather than just allowing a width and height as input. Perhaps the input parameters could be the ratio and width, and the height is calculated dynamically. The reason being is that for someone not aware that the numbers are chosen specifically for SEO benefits, it might be very easy for them to change the numbers without realising the impact.

Reply:
@Ridhwana that's a good thought in terms of communicating intent. I still think passing in the actual h/w values provides more flexibility here.

But maybe this array could be extracted as a named method like `seo_optimized_images` so it's clear enough. And then even a comment that links to the Google guide.

In general with all of these json ld methods it might be useful to have a comment that includes the url of the guide because that's the context that one would need to really understand how and why to modify this.

@juliannatetreault I'd still vote to ship this and consider refactoring after.

Reply:
Thanks so much, @Ridhwana ! I'm happy to hear that you're learning alongside me. 😄

Regarding your thought: I am happy to look into the possibility of using a calculation for this to lessen confusion for contributors and the team, but I was also thinking maybe me adding a comment in the code to help alleviate the confusion in the meantime could be a useful solution?

Reply:
That's a good suggestion @benhalpern and I can open an issue and the card to the project board so that I can get this shipped and refactor later. 😄

Reply:
This sounds good 🎉its definitely not a blocker.